File: Logistics.Data/HandReceiptItem.cs
namespace Logistics.Data
{
    public class HandReceiptItem
    {
        public int Id { get; set; }
        public int HandReceiptId { get; set; }
        public int ItemId { get; set; }
        public int Quantity { get; set; }

        public HandReceipt HandReceipt { get; set; }
        public NsnItem NsnItem { get; set; }
    }
}

File: Logistics.Web/ClientApp/src/app/models/hand-receipt-item.ts
import { HandReceipt } from './hand-receipt';
import { NsnItem } from './item';

export class HandReceiptItem {
  id: number;
  handReceiptItem: number;
  itemId: number;
  quantity: number;
  handReceipt: HandReceipt;
  nsnItem: NsnItem;
}
